tutorials/io/experimental/rfile001_basics.C

```diff
@@ -0,0 +1,53 @@
+// \file tutorials/io/experimental/rfile.py
+// \ingroup Base ROOT7
+// \author Giacomo Parolini <giacomo.parolini@cern.ch>
+// \date 2025-06-11
+// \warning This is part of the ROOT 7 prototype! It will change without notice. It might trigger earthquakes. Feedback
```

Comment:
Consider replacing `//` with `///`. I'm not sure if doxygen would interpret this section otherwise.

Once confirmed that doxygen runs, consider adding:
```suggestion
/// \macro_code
/// \macro_output
// \warning This is part of the ROOT 7 prototype! It will change without notice. It might trigger earthquakes. Feedback
```
This will generate a page with the code, and also run the tutorial and show the output of the program.